ios/AppLovinMAXNativeAdView.m

```diff
@@ -221,7 +221,7 @@ - (void)setIconView:(NSNumber *)tag
         if ( !icon.URL && icon.image )
         {
             RCTImageView *iconImageView = (RCTImageView *) view;
-            iconImageView.defaultImage = icon.image;
+            [iconImageView performSelector: @selector(setImage:) withObject: icon.image];
```

Comment:
Actually kind of dangerous if RN changes underlying API - to be completely save I'd do `respondsToSelector:` first. In the `else` case - log an error.

Also, can you check if there are any Xcode warnings this introduces?

Reply:
Thanks, I actually wondered if it would be okay to access a private method via `selector`.

I don't see any warnings from this change.